### ISSUE TITLE
Coding of missing values

### ISSUE THREAD

**Issue by ju-rgen** [Open-Systems-Pharmacology/OSPSuite.Core#828]
Because in often used NONMEM missing values cannot be coded by NAN, but by some special numeric value chosen by convention, e.g. -1E-99 (or -999.999 or -0.999) there should be an option to define a numeric value for interpretation as "missing value".

**Comment by msevestre** [comment]
@ju-rgen Where will this be used? When importing data?

**Comment by ju-rgen** [comment]
Yes. During import the coding should be resolved to the OSP internal representation of missing values.
